plugins/orchestrator-backend/src/service/SonataFlowService.ts

```diff
@@ -587,7 +572,7 @@ export class SonataFlowService {
       uri: await this.fetchWorkflowUri(workflowId),
       lastTriggeredMs: lastTriggered.getTime(),
       lastRunStatus,
-      type: this.extractWorkflowType(definition),
+      type: getWorkflowCategory(definition),
```

Comment:
```suggestion
      category: getWorkflowCategory(definition),
```

In order to avoid future confusion, could you please rename `type` -> `category` in the `WorkflowOverview` interface?

Reply:
Addressed, thanks for the suggestion.

@jkilzi I updated the render in the UI, can you please double check?